awebox/viz/output.py

```diff
@@ -565,25 +518,60 @@ def plot_aero_forces(solution_dict, cosmetics, fig_num, reload_dict):
     fig.canvas.draw()
 
 # def plot_output(solution_dict, cosmetics, fig_num, reload_dict): #todo: fix output plot!
```

Comment:
I suppose you can remove this todo

Reply:
ok. :) not sure if there was something else implied by this comment, but...